sdk/purview/Azure.Analytics.Purview.Share/README.md

```diff
@@ -59,15 +59,235 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 The following section shows you how to initialize and authenticate your client and get a sent share.
 
-### Get a Sent Share
+### Create sent share
 
 ```C#
 var credential = new DefaultAzureCredential();
-var client = new SentSharesClient(new Uri("https://<my-account-name>.purview.azure.com"), credential);
+var endpoint = "https://<my-account-name>.purview.azure.com";
```

Comment:
All code samples should be in the form of snippets, so that they can be compiled, and smoke tested nightly.  More context can be found in [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets) of the contributing guide.  

Examples:
- [Event Hubs README inclusion](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/README.md#inspect-an-event-hub)
- [Event Hubs snippet source](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/tests/Snippets/ReadMeSnippetsLiveTests.cs#L21)
- [KeyVault README inclusion](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md#create-a-secret)
- [KeyVault snippet source](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/keyvault/Azure.Security.KeyVault.Secrets/tests/samples/SampleSnippets.cs#L45)

Reply:
Thank you @jsquire